src/helpers/shared/user-login.js

```diff
@@ -1,9 +1,36 @@
 /* eslint-disable no-unused-vars */
-import axiosInstance from '@redhat-cloud-services/frontend-components-utilities/interceptors';
+import axios from 'axios';
+import {
+  authInterceptor,
+  responseDataInterceptor,
+  interceptor401,
+  interceptor500,
+  errorInterceptor,
+} from '@redhat-cloud-services/frontend-components-utilities/interceptors';
```

Comment:
Is there a reason why not to import the instance here and just pass in `interceptor403`? You are effectively doing the same thing, but on more lines.

Reply:
Yes. Because the order of interceptors matters. The common error interceptor changes the error payload. It is stripping some very important information - the status:
```js
throw { ...errObject.response.data, statusText: errObject.response.statusText };
```
Which makes it impossible to get the status code from the error payload. If we could change the order of interceptors, I would use the default instance.

Reply:
Ah I see, we might want to create `instanceBuilder` which would take an array of `request` interceptors whose will be used before using `authInterceptor` and an array of `response` interceptors whose will be used before using `interceptor401``

Reply:
Well, the other interceptors don't really matter because they throw the original error. Only the last one is changing the payload.